crates/orchestrator/run-config.toml

```diff
@@ -39,22 +39,299 @@ seed = [
 ]
 start_delay_seconds = 0
 cdn_marshal_address = "127.0.0.1:9000"
+public_keys = [
+    [
```

Comment:
could we format this to a single line? (especially if we might add more keys to this list)

happy to do the formatting myself!